README.md

```diff
@@ -1,9 +1,38 @@
 # codex-discord-bot
 CODEX bot for the CODEX Discord channel
+[![Gitter](https://badges.gitter.im/codex-discord-bot/community.svg)](https://gitter.im/codex-discord-bot/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 # MAINTAINERS
 - [Debashish Mishra](https://github.com/Zanark)
 
-# CONTRIBUTORS
+# Requirements
```

Comment:
Don't move the Contributors list, add the new stuff after the Contributors list.